decision_service/include/live_model.h

```diff
@@ -1,24 +1,51 @@
 #pragma once
+#include "ranking_response.h"
+#include "err_constants.h"
+#include "factory_resolver.h"
+
 namespace reinforcement_learning {
 
-  // Forward declarations
-  class live_model_impl;
-  class ranking_response;
-  class api_status;
-  namespace utility { class config_collection; }
-  ////
+  //// Forward declarations ////////
+  class live_model_impl;          //
+  class ranking_response;         //
+  class api_status;               //
+                                  //
+  namespace model_management {    //
+    class i_data_transport;       //
+    class i_model;                //
+  }                               //
+                                  //
+  namespace utility {             //
+    class config_collection;      //
+  }                               //
+  //////////////////////////////////
 
-	//decision service client
+	// Reinforcement learning client
 	class live_model {
 
 	public:
-    using error_fn = void(*)(const api_status&, void*);
-    explicit live_model(const utility::config_collection& config, error_fn fn = nullptr, void* err_context = nullptr);
+    // Type declarations
+    using error_fn            = void(*)(const api_status&, void*);
+    using transport_factory_t = utility::object_factory<model_management::i_data_transport>;
```

Comment:
nit: this use of alignment feels out of place

Reply:
Changed.